src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -125,6 +126,8 @@ The first one will be picked, you should probably set only one default value`
       );
     }
 
+    const defaultValue = find(userItems, item => item.default === true);
```

Comment:
what is item.default? (Always thought default was a reserved word in JS)

Reply:
It's a boolean flag that expresses the default value of the hitsPerPage. It is a reserved keyword indeed https://docs.microsoft.com/en-us/scripting/javascript/reference/javascript-reserved-words but we're safe https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Reserved_word_usage